workers/docker-worker/src/lib/task_listener.js

```diff
@@ -137,54 +119,102 @@ class TaskListener extends EventEmitter {
       this.runtime.monitor,
     );
     // Do not claim tasks if not enough resources are available
-    if (exceedsThreshold) {return;}
+    if (exceedsThreshold) {
+      debug('not calling claimWork because not enough disk space');
```

Comment:
Thanks for adding this

Reply:
Note it's only debug, but it's at least useful in testing.  There was another PR to do a better job of logging out loud when the devices don't initialize.